stylus/include/TheBlog.styl

```diff
@@ -0,0 +1,35 @@
+.TheBlog
+	margin-bottom: 3rem
+	margin-top: 8rem
+	text-align: center
+	>h1
+		font-family: Arvo
+		margin-bottom: 3rem
+	>p
+		margin-right: 33%
+		margin-left: 33%
+	.calendar-images
+		display: flex
+		flex-wrap: wrap
+		justify-content: center
+		margin-left: 3rem
+		margin-right: 3rem
+		margin-top: 3rem
+		>div
+			background: $Mustard
+			flex: 1
```

Comment:
Yeah, I'm officially now of the opinion that we don't need a `flex` style for these. Like with the farmers markets on the Where To Find Us page, look what happens when one image is left over:
![screenshot at 2018-07-09 08 06 27](https://user-images.githubusercontent.com/10747047/42458765-3e0009c4-834f-11e8-9fbe-b19c8a0603f2.png)
We don't want the images to grow to take up left over space. We want them all the same width. So just let `flex-basis` take care of that and remove `flex`. BTW I'd probably take `flex-basis` down to 20rem. The images are just a tad bit too large in my opinion.